raiden/tests/api/test_api.py

```diff
@@ -434,10 +444,12 @@ def test_api_open_channel_invalid_input(
     partner_address = '0x61c808d82a3ac53231750dadc13c777b59310bd9'
     token_address = '0xea674fdde714fd979de3edf0f56aa9716b898ec8'
     settle_timeout = NETTINGCHANNEL_SETTLE_TIMEOUT_MIN - 1
+    reveal_timeout = 30
```

Comment:
here and in other places where you use `30` shouldn't `DEFAULT_REVEAL_TIMEOUT` be used?

Reply:
@LefterisJP most of those tests just use constants as testing parameters, but I can use `DEFAULT_REVEAL_TIMEOUT` too, of course.